pkg/nativeimgutil/fuzz_test.go

```diff
@@ -0,0 +1,33 @@
+package nativeimgutil
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func FuzzConvertToRaw(f *testing.F) {
+	f.Fuzz(func(t *testing.T, imgData []byte, withBacking, specifySize bool) {
+		if len(imgData) < 3 {
+			return
+		}
+		srcPath := filepath.Join(t.TempDir(), "src.img")
+		destPath := filepath.Join(t.TempDir(), "dest.img")
+		err := os.WriteFile(srcPath, imgData, 0o600)
+		if err != nil {
+			return
+		}
+		if specifySize {
+			f, err := os.Stat(srcPath)
+			if err != nil {
+				return
+			}
+			fileSize := f.Size()
+			//nolint:errcheck // The test doesn't check the return value
+			ConvertToRaw(srcPath, destPath, &fileSize, withBacking)
+		} else {
+			//nolint:errcheck // The test doesn't check the return value
+			ConvertToRaw(srcPath, destPath, nil, withBacking)
```

Comment:
If you edit the PR anyways, please use `_` instead of the `//nolint` directives.